packages/backend-app-api/src/services/implementations/httpRouter/httpRouterFactory.ts

```diff
@@ -27,7 +27,7 @@ export interface HttpRouterFactoryOptions {
   /**
    * A callback used to generate the path for each plugin, defaults to `/api/{pluginId}`.
    */
-  getPath(pluginId: string): string;
+  getPath?(pluginId: string): string;
```

Comment:
Feel weird that this would be required in the options even though we provide a default for it.